# English (Australia) translation for testdrive
# Copyright (c) 2011 Rosetta Contributors and Canonical Ltd 2011
# This file is distributed under the same license as the testdrive package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2011.
#
msgid ""
msgstr ""
"Project-Id-Version: testdrive\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2013-08-02 08:14+1000\n"
"PO-Revision-Date: 2013-08-10 08:09+0000\n"
"Last-Translator: Jackson Doak <Doak.jackson@gmail.com>\n"
"Language-Team: English (Australia) <en_AU@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2013-08-10 20:12+0000\n"
"X-Generator: Launchpad (build 16723)\n"

#: ../testdrive-gtk.desktop.in.h:1
msgid "Test Drive an Ubuntu ISO"
msgstr "Test Drive an Ubuntu ISO"

#: ../testdrive-gtk.desktop.in.h:2
msgid "Download and run an Ubuntu ISO in a Virtual Machine"
msgstr "Download and run an Ubuntu ISO in a Virtual Machine"

#: ../testdrive-gtk.desktop.in.h:3
msgid "ISO;Testing;Ubuntu;Daily;VM;Virtual Machine;zsync;rsync;Testdrive"
msgstr "ISO;Testing;Ubuntu;Daily;VM;Virtual Machine;zsync;rsync;Testdrive"

#: ../bin/testdrive-gtk.py:49
msgid "Other"
msgstr "Other"

#. ##################################################################
#. ####### Obtaining the settings from the Preferences Class ########
#. ##################################################################
#: ../bin/testdrive-gtk.py:139
msgid "Instancing Preferences..."
msgstr "Instancing Preferences..."

#: ../bin/testdrive-gtk.py:222
#, python-format
msgid ""
"Checking available ISOs at the ISO tracker [http://iso.qa.ubuntu.com] every "
"%s seconds"
msgstr ""
"Checking available ISOs at the ISO tracker [http://iso.qa.ubuntu.com] every "
"%s seconds"

#: ../bin/testdrive-gtk.py:227
msgid "Unable to check the ISO tracker"
msgstr "Unable to check the ISO tracker"

#: ../bin/testdrive-gtk.py:265
#, python-format
msgid "<b>Release:</b> %s"
msgstr "<b>Release:</b> %s"

#: ../bin/testdrive-gtk.py:266
#, python-format
msgid "<b>ISO Repository:</b> http://%s.ubuntu.com/"
msgstr "<b>ISO Repository:</b> http://%s.ubuntu.com/"

#: ../bin/testdrive-gtk.py:306
msgid "Add an ISO to TestDrive"
msgstr "Add an ISO to TestDrive"

#: ../bin/testdrive-gtk.py:322
msgid ""
"Unable to open because 'testdrive' is not installed.\n"
"Please install testdrive: \n"
"\n"
"sudo apt-get install testdrive-cli"
msgstr ""
"Unable to open because 'testdrive' is not installed.\n"
"Please install testdrive: \n"
"\n"
"sudo apt-get install testdrive-cli"

#: ../bin/testdrive-gtk.py:326
msgid "TestDrive an ISO or Disk Image"
msgstr "TestDrive an ISO or Disk Image"

#. ##################################################################
#. ## Obtains the command for the Sync process based on Protocol ####
#. ##################################################################
#: ../bin/testdrive-gtk.py:401
msgid "Obtaining the sync protocol for the specified ISO..."
msgstr "Obtaining the sync protocol for the specified ISO..."

#: ../bin/testdrive-gtk.py:404 ../bin/testdrive.py:375
#, python-format
msgid "Unsupported protocol [%s]"
msgstr "Unsupported protocol [%s]"

#. ##################################################################
#. #### Obtains the Virtualization Method, if not shows warnings ####
#. ##################################################################
#. Choose the virtualization engine
#: ../bin/testdrive-gtk.py:415
msgid "Obtaining the virtualization method..."
msgstr "Obtaining the virtualisation method..."

#: ../bin/testdrive-gtk.py:419
msgid "Your CPU supports KVM acceleration; please install KVM"
msgstr "Your CPU supports KVM acceleration; please install KVM"

#: ../bin/testdrive-gtk.py:420
msgid ""
"Your CPU supports KVM acceleration; please install KVM:\n"
"\n"
"sudo apt-get install qemu-kvm"
msgstr ""
"Your CPU supports KVM acceleration; please install KVM:\n"
"\n"
"sudo apt-get install qemu-kvm"

#: ../bin/testdrive-gtk.py:424
msgid ""
"Your CPU does not support acceleration; run kvm-ok for more information; "
"then install VBox"
msgstr ""
"Your CPU does not support acceleration; run kvm-ok for more information; "
"then install VBox"

#: ../bin/testdrive-gtk.py:425
msgid ""
"Your CPU does not support acceleration; run kvm-ok for more information;\n"
"then please install VirtualBox\n"
"\n"
"kvm-ok\n"
"sudo apt-get install virtualbox-ose"
msgstr ""
"Your CPU does not support acceleration; run kvm-ok for more information;\n"
"then please install VirtualBox\n"
"\n"
"kvm-ok\n"
"sudo apt-get install virtualbox-ose"

#. ##################################################################
#. ### Obtains the list of ISO available from the iso list cache ####
#. ##################################################################
#. Try to retrieve the ISO list from the cache
#: ../bin/testdrive-gtk.py:438 ../bin/testdrive.py:348
msgid "Retrieving the Ubuntu ISO list from cache..."
msgstr "Retrieving the Ubuntu ISO list from cache..."

#: ../bin/testdrive-gtk.py:442 ../bin/testdrive-gtk.py:443
#: ../bin/testdrive.py:352
msgid "Unable to retrieve the Ubuntu ISO list from cache..."
msgstr "Unable to retrieve the Ubuntu ISO list from cache..."

#: ../bin/testdrive-gtk.py:563 ../bin/testdrive-gtk.py:598
#: ../bin/testdrive-gtk.py:905
#, python-format
msgid "<i>     CACHE: [%s]</i>"
msgstr "<i>     CACHE: [%s]</i>"

#: ../bin/testdrive-gtk.py:565 ../bin/testdrive-gtk.py:600
#: ../bin/testdrive-gtk.py:907
msgid "<i>     CACHE: [empty]</i>"
msgstr "<i>     CACHE: [empty]</i>"

#: ../bin/testdrive-gtk.py:628
msgid "<b><i>There are no Other ISOs yet...</i></b>"
msgstr "<b><i>There are no Other ISOs yet...</i></b>"

#: ../bin/testdrive-gtk.py:631
msgid "<b><i>There are no ISOs for this architecture yet...</i></b>"
msgstr "<b><i>There are no ISOs for this architecture yet...</i></b>"

#. Add Button
#: ../bin/testdrive-gtk.py:665
msgid "Add ISO"
msgstr "Add ISO"

#. Create Buttons
#: ../bin/testdrive-gtk.py:672
msgid "Create USB Disk"
msgstr "Create USB Disk"

#: ../bin/testdrive-gtk.py:677
msgid "Sync"
msgstr "Sync"

#: ../bin/testdrive-gtk.py:682
msgid "Launch"
msgstr "Launch"

#: ../bin/testdrive-gtk.py:736 ../bin/testdrive.py:264
msgid "Using KVM for virtual machine hosting..."
msgstr "Using KVM for virtual machine hosting..."

#: ../bin/testdrive-gtk.py:739 ../bin/testdrive.py:266
msgid "Using VirtualBox for virtual machine hosting..."
msgstr "Using VirtualBox for virtual machine hosting..."

#: ../bin/testdrive-gtk.py:742 ../bin/testdrive.py:268
msgid "Using Parallels Desktop for virtual machine hosting..."
msgstr "Using Parallels Desktop for virtual machine hosting..."

#: ../bin/testdrive-gtk.py:796
msgid "sync_iso: Thread is executing..."
msgstr "sync_iso: Thread is executing..."

#: ../bin/testdrive-gtk.py:817
msgid "launch_iso: Thread is executing or syncing..."
msgstr "launch_iso: Thread is executing or syncing..."

#: ../bin/testdrive-gtk.py:824
msgid ""
"No ISO has been selected.\n"
"\n"
"Please select an ISO to create an USB Startup Disk."
msgstr ""
"No ISO has been selected.\n"
"\n"
"Please select an ISO to create an USB Startup Disk."

#: ../bin/testdrive-gtk.py:829
msgid ""
"More than 1 ISO has been selected.\n"
"\n"
"Please select only 1 ISO to continue!"
msgstr ""
"More than 1 ISO has been selected.\n"
"\n"
"Please select only 1 ISO to continue!"

#: ../bin/testdrive-gtk.py:834
msgid ""
"The specified ISO does not exist!\n"
"\n"
"Please, synchronize the ISO to continue."
msgstr ""
"The specified ISO does not exist!\n"
"\n"
"Please, synchronise the ISO to continue."

#: ../bin/testdrive-gtk.py:841
msgid "Unable to launch USB Creator!"
msgstr "Unable to launch USB Creator!"

#: ../bin/testdrive-gtk.py:867
#, python-format
msgid " Downloading %s (0%%)"
msgstr " Downloading %s (0%%)"

#: ../bin/testdrive-gtk.py:887
#, python-format
msgid " Downloading %s (%s%%)"
msgstr " Downloading %s (%s%%)"

#: ../bin/testdrive-gtk.py:936
msgid "    Configuring Virtual Machine..."
msgstr "    Configuring Virtual Machine..."

#: ../bin/testdrive-gtk.py:942
msgid "     Running Virtual Machine..."
msgstr "     Running Virtual Machine..."

#: ../bin/testdrive-gtk.py:991
msgid "Show debug messages"
msgstr "Show debug messages"

#: ../bin/testdrive.py:52
msgid ""
"\n"
"Welcome to Testdrive!\n"
msgstr ""
"\n"
"Welcome to Testdrive!\n"

#: ../bin/testdrive.py:60
#, python-format
msgid "     +-cache--> [%s] %s"
msgstr "     +-cache--> [%s] %s"

#: ../bin/testdrive.py:63
#, python-format
msgid "  %d. Other (prompt for ISO URL)"
msgstr "  %d. Other (prompt for ISO URL)"

#: ../bin/testdrive.py:65
msgid ""
"\n"
"Select an image to testdrive [1]: "
msgstr ""
"\n"
"Select an image to testdrive [1]: "

#: ../bin/testdrive.py:74
msgid ""
"\n"
"ERROR: Invalid input\n"
msgstr ""
"\n"
"ERROR: Invalid input\n"

#: ../bin/testdrive.py:77
msgid ""
"\n"
"Enter an ISO URL to testdrive: "
msgstr ""
"\n"
"Enter an ISO URL to testdrive: "

#: ../bin/testdrive.py:84
msgid ""
"\n"
"ERROR: Invalid selection\n"
msgstr ""
"\n"
"ERROR: Invalid selection\n"

#: ../bin/testdrive.py:88
#, python-format
msgid ""
"\n"
"%s\n"
msgstr ""
"\n"
"%s\n"

#: ../bin/testdrive.py:102 ../testdrivegtk/AddOtherTestdrivegtkDialog.py:155
#, python-format
msgid "Invalid ISO URL [%s]"
msgstr "Invalid ISO URL [%s]"

#: ../bin/testdrive.py:144
#, python-format
msgid ""
"Command failed\n"
"    `%s`"
msgstr ""
"Command failed\n"
"    `%s`"

#: ../bin/testdrive.py:151
msgid "Launching Virtual Machine using CURSES as screen mode"
msgstr "Launching Virtual Machine using CURSES as screen mode"

#: ../bin/testdrive.py:196
msgid "user configuration file (overriding default values"
msgstr "user configuration file (overriding default values"

#: ../bin/testdrive.py:198
msgid "print version and system data, and exit"
msgstr "print version and system data, and exit"

#: ../bin/testdrive.py:200
msgid "get ISO image from this URL location"
msgstr "get ISO image from this URL location"

#: ../bin/testdrive.py:202
msgid "try to launch usb-creator for further testing"
msgstr "try to launch usb-creator for further testing"

#: ../bin/testdrive.py:204
msgid "hardcode Ubuntu RELEASE codename"
msgstr "hardcode Ubuntu RELEASE codename"

#: ../bin/testdrive.py:206
msgid ""
"hardcode Ubuntu flavor. Available Flavors:\n"
"             "
"ubuntu/kubuntu/xubuntu/edubuntu/mythbuntu/ubuntustudio/lubuntu/ubuntukylin"
msgstr ""
"hardcode Ubuntu flavor. Available Flavors:\n"
"             "
"ubuntu/kubuntu/xubuntu/edubuntu/mythbuntu/ubuntustudio/lubuntu/ubuntukylin"

#: ../bin/testdrive.py:209
msgid ""
"hardcode Ubuntu repository from where to obtain ISOs:\n"
"             releases/cdimage/cloud-daily/cloud-releases"
msgstr ""
"hardcode Ubuntu repository from where to obtain ISOs:\n"
"             releases/cdimage/cloud-daily/cloud-releases"

#: ../bin/testdrive.py:212
msgid ""
"displays the Virtual Machine in the shell. Only valid for Cloud images."
msgstr ""
"displays the Virtual Machine in the shell. Only valid for Cloud images."

#: ../bin/testdrive.py:217
#, python-format
msgid "version passed: %s"
msgstr "version passed: %s"

#: ../bin/testdrive.py:221
#, python-format
msgid "testdrive %s"
msgstr "testdrive %s"

#: ../bin/testdrive.py:233
#, python-format
msgid "config passed: %s"
msgstr "config passed: %s"

#: ../bin/testdrive.py:243
#, python-format
msgid "Trying config in %s"
msgstr "Trying config in %s"

#: ../bin/testdrive.py:247
#, python-format
msgid "Using configuration in %s"
msgstr "Using configuration in %s"

#: ../bin/testdrive.py:249
#, python-format
msgid "Invalid configuration [%s]"
msgstr "Invalid configuration [%s]"

#: ../bin/testdrive.py:257
msgid ""
"Your CPU supports KVM acceleration; please install KVM:\n"
"            sudo apt-get install qemu-kvm"
msgstr ""
"Your CPU supports KVM acceleration; please install KVM:\n"
"            sudo apt-get install qemu-kvm"

#: ../bin/testdrive.py:260
msgid ""
"Your CPU does not support acceleration; run kvm-ok for more information; "
"then please install VirtualBox:\n"
"            kvm-ok\n"
"            sudo apt-get install virtualbox-ose"
msgstr ""
"Your CPU does not support acceleration; run kvm-ok for more information; "
"then please install VirtualBox:\n"
"            kvm-ok\n"
"            sudo apt-get install virtualbox-ose"

#: ../bin/testdrive.py:284
msgid ""
"Launching Cloud images only works with KVM. Please switch your "
"virtualization method..."
msgstr ""
"Launching Cloud images only works with KVM. Please switch your "
"virtualisation method..."

#: ../bin/testdrive.py:335
#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:333
#, python-format
msgid "Obtaining Ubuntu ISO list from %s..."
msgstr "Obtaining Ubuntu ISO list from %s..."

#: ../bin/testdrive.py:339
#, python-format
msgid "ERROR: Could not obtain the Ubuntu ISO list from %s..."
msgstr "ERROR: Could not obtain the Ubuntu ISO list from %s..."

#: ../bin/testdrive.py:345
#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:345
msgid "Unable to update the Ubuntu ISO list cache..."
msgstr "Unable to update the Ubuntu ISO list cache..."

#. BUG: should check disk space availability in CACHE dir
#. Update the cache
#: ../bin/testdrive.py:371
msgid "Syncing the specified ISO..."
msgstr "Syncing the specified ISO..."

#: ../bin/testdrive.py:378 ../testdrivegtk/AddOtherTestdrivegtkDialog.py:167
#, python-format
msgid "ISO not found at [%s]"
msgstr "ISO not found at [%s]"

#. ##################
#. # Launch the VM ##
#. ##################
#: ../bin/testdrive.py:401
msgid "Validating Virtualization Method...."
msgstr "Validating Virtualisation Method...."

#: ../bin/testdrive.py:405
#, python-format
msgid "Unable to validate Virtualization Method [%s]"
msgstr "Unable to validate Virtualisation Method [%s]"

#: ../bin/testdrive.py:407
msgid "Setting up Virtual Machine..."
msgstr "Setting up Virtual Machine..."

#: ../bin/testdrive.py:411
msgid "Unable to setup Virtual Machine"
msgstr "Unable to setup Virtual Machine"

#: ../bin/testdrive.py:413
msgid "Launching Virtual Machine..."
msgstr "Launching Virtual Machine..."

#: ../bin/testdrive.py:421
msgid "Unable to launch Virtual Machine"
msgstr "Unable to launch Virtual Machine"

#: ../bin/testdrive.py:425
#, python-format
msgid "Cleaning up disk image [%s]..."
msgstr "Cleaning up disk image [%s]..."

#. Remind about cache cleanup
#: ../bin/testdrive.py:428
msgid "You may wish to clean up the cache directory..."
msgstr "You may wish to clean up the cache directory..."

#: ../bin/testdrive.py:429
#, python-format
msgid "      %s and %s"
msgstr "      %s and %s"

#: ../bin/testdrive.py:436
msgid ""
"\n"
"Launch USB Startup Disk Creator for further testing of this ISO? [y/N] "
msgstr ""
"\n"
"Launch USB Startup Disk Creator for further testing of this ISO? [y/N] "

#: ../bin/testdrive.py:440
msgid ""
"\n"
"Press <enter> to exit..."
msgstr ""
"\n"
"Press <enter> to exit..."

#: ../testdrivegtk/AddOtherTestdrivegtkDialog.py:135
msgid "Please insert a Description"
msgstr "Please insert a Description"

#: ../testdrivegtk/AddOtherTestdrivegtkDialog.py:140
msgid "Please insert a URL"
msgstr "Please insert a URL"

#: ../testdrivegtk/AddOtherTestdrivegtkDialog.py:171
msgid "No sync protocol has been selected, please select one."
msgstr "No sync protocol has been selected, please select one."

#: ../testdrivegtk/AddOtherTestdrivegtkDialog.py:196
msgid "Description"
msgstr "Description"

#: ../testdrivegtk/AddOtherTestdrivegtkDialog.py:227
msgid "Select Protocol:"
msgstr "Select Protocol:"

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:36
#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:37
msgid "Other..."
msgstr "Other..."

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:72
msgid "TestDrive Preferences"
msgstr "TestDrive Preferences"

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:98
msgid "finish_initialization()"
msgstr "finish_initialization()"

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:138
msgid "Select Repository:"
msgstr "Select Repository:"

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:149
#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:490
msgid "Select Release:"
msgstr "Select Release:"

#. Load config files for local variables
#. self.load_config_files(file)
#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:217
#, python-format
msgid "Reading config file: [%s]"
msgstr "Reading config file: [%s]"

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:219
#, python-format
msgid "Unable to load config file [%s]"
msgstr "Unable to load config file [%s]"

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:276
#, python-format
msgid " of %s available."
msgstr " of %s available."

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:337
#, python-format
msgid "Could not obtain the Ubuntu ISO list from %s..."
msgstr "Could not obtain the Ubuntu ISO list from %s..."

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:341
msgid "Updating the Ubuntu ISO list cache..."
msgstr "Updating the Ubuntu ISO list cache..."

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:349
msgid "get_preferences()"
msgstr "get_preferences()"

#: ../testdrivegtk/PreferencesTestdrivegtkDialog.py:472
#, python-format
msgid "Unable to clean up files from [%s]"
msgstr "Unable to clean up files from [%s]"
